main.go

```diff
@@ -13,6 +13,8 @@ import (
 func main() {
 	outputFilePtr := flag.String("outputFile", "validators.go", "output file name")
 	needValidatableCheckPtr := flag.Bool("needValidatableCheck", true, "check struct on Validatable before calling Validate()")
+	excludeRegexp := flag.String("excludeRegexp", `(client\.go|client_mock\.go)`,
+		"comma separated list of file names that generator should exclude, e.g. `client.go;client_mock.go`")
```

Comment:
comma separated? examples has `;` . And maybe rename flag to excludeFiles or excludeList ? it is not regexp from client view.